arches/settings_utils.py

```diff
@@ -191,23 +191,24 @@ def generate_frontend_configuration():
                             "arches",
                             "*",
                         )
-                    ],
-                    **{
-                        os.path.join("@", path_name, "*"): [
-                            os.path.join(
-                                ".",
-                                os.path.relpath(path, os.path.join(base_path, "..")),
-                                "src",
-                                path_name,
-                                "*",
-                            )
-                        ]
-                        for path_name, path in path_lookup.items()
-                    },
+                    ]
                 }
             },
         }
 
+        if settings.APP_NAME != "Arches":
```

Comment:
This removes arches applications from the list of tsconfigs completely, which prevents useful type checking. Is there another solution?